Assets/HoloToolkit/Input/Tests/Scenes/OverrideFocusedObjectTest.unity

```diff
@@ -1,478 +0,0 @@
-%YAML 1.1
-%TAG !u! tag:unity3d.com,2011:
```

Comment:
Ahh, perhaps I misunderstood.  I was under the impression that the OverrideFocusedObject was introduced as a temporary workaround for a pointer selection bug in the past.  My memory is just hazy though.  Did I remember that wrong?

Using a global listener or a modal input handler seems like the right way to handle override scenarios, was there some scenario they couldn't satisfy?